lib/ops/ops.go

```diff
@@ -759,35 +762,49 @@ func (r RotateSecretsRequest) SiteKey() SiteKey {
 	}
 }
 
-// RotatePlanetConfigRequest is a request to rotate server's planet configuration package
-type RotatePlanetConfigRequest struct {
-	// AccountID is the account id of the local cluster
-	AccountID string `json:"account_id"`
-	// ClusterName is the local cluster name
-	ClusterName string `json:"cluster_name"`
-	// OperationID is the id of the operation
-	OperationID string `json:"operation_id"`
+// Check validates this request
+func (r RotateTeleportConfigRequest) Check() error {
```

Comment:
nit: I believe we use CheckAndSetDefaults as a standard name in other locations in the codebase, even if Defaults are not being set.